bindings/python/IK/tests/test_QP_inverse_kinematics.py

```diff
@@ -166,6 +166,40 @@ def test_joint_tracking_task():
     joint_velocities = [np.random.uniform(-0.5,0.5) for _ in range(kindyn_desc.kindyn.get_nr_of_dofs())]
     assert joint_tracking_task.set_set_point(joint_position=joint_values,joint_velocity=joint_velocities)
 
+def test_angular_momentum_task():
+
+    # retrieve the model
+    model_url = 'https://raw.githubusercontent.com/robotology/icub-models/master/iCub/robots/iCubGazeboV2_5/model.urdf'
```

Comment:
Perhaps you may want to get a fixed release to avoid that a change in icub-models breaks this test?

Reply:
I think that as soon as we can start using `pybind11` `iDynTree` bindings we can switch to `iDynTree.get_random_chain(10)`

Reply:
In that case, we need to remind to set a fixed seed.